upgrade/UPGRADE-v8.1.0-dev.md

```diff
@@ -844,6 +844,12 @@ There you can find links to upgrade notes for other versions too.
         +     }
         + },
         ```
+- message error with long words ([#1561](https://github.com/shopsys/shopsys/pull/1561))
```

Comment:
Please change the title of this upgrade not. Its not about error messages with long words